rental.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+
+class Rental
+  attr_accessor :person, :book, :date
+
+  def initialize(person, book, date)
+    @person = person
+    @book = book
+    @date = date
+  end
```

Comment:
The rental should add itself to the @book and @person to help achieve the `belongs-to` associations. Kindly consider doing something like below for both books and person to achieve that. :point_down:  (this is just a code sample)

             @book = book
             book.rentals << self